MekHQ/src/mekhq/gui/dialog/CustomizePersonDialog.java

```diff
@@ -1315,36 +1295,39 @@ public int getAge() {
     }
 
     private void backgroundChanged() {
-        int pheno = choicePheno.getSelectedIndex();
         boolean clanner = chkClan.isSelected();
         clearAllPhenotypeBonuses();
         if (clanner) {
-            switch(pheno) {
-                case Person.PHENOTYPE_MW:
-                    skillBonus.get(SkillType.S_GUN_MECH).setValue(1);
-                    skillBonus.get(SkillType.S_PILOT_MECH).setValue(1);
-                    break;
-                case Person.PHENOTYPE_AERO:
-                    skillBonus.get(SkillType.S_GUN_AERO).setValue(1);
-                    skillBonus.get(SkillType.S_PILOT_AERO).setValue(1);
-                    skillBonus.get(SkillType.S_GUN_JET).setValue(1);
-                    skillBonus.get(SkillType.S_PILOT_JET).setValue(1);
-                    skillBonus.get(SkillType.S_GUN_PROTO).setValue(1);
-                    break;
-                case Person.PHENOTYPE_BA:
-                    skillBonus.get(SkillType.S_GUN_BA).setValue(1);
-                    break;
-                case Person.PHENOTYPE_VEE:
-                    skillBonus.get(SkillType.S_GUN_VEE).setValue(1);
-                    skillBonus.get(SkillType.S_PILOT_GVEE).setValue(1);
-                    skillBonus.get(SkillType.S_PILOT_NVEE).setValue(1);
-                    skillBonus.get(SkillType.S_PILOT_VTOL).setValue(1);
-                    break;
+            final Phenotype phenotype = (Phenotype) choicePhenotype.getSelectedItem();
+            if (phenotype != null) {
+                // TODO : Windchild should there be more bonus' here for the other phenotypes?
+                switch (phenotype) {
+                    case MECHWARRIOR:
+                        skillBonus.get(SkillType.S_GUN_MECH).setValue(1);
+                        skillBonus.get(SkillType.S_PILOT_MECH).setValue(1);
+                        break;
+                    case AEROSPACE:
+                        skillBonus.get(SkillType.S_GUN_AERO).setValue(1);
+                        skillBonus.get(SkillType.S_PILOT_AERO).setValue(1);
+                        skillBonus.get(SkillType.S_GUN_JET).setValue(1);
+                        skillBonus.get(SkillType.S_PILOT_JET).setValue(1);
+                        skillBonus.get(SkillType.S_GUN_PROTO).setValue(1);
+                        break;
+                    case ELEMENTAL:
+                        skillBonus.get(SkillType.S_GUN_BA).setValue(1);
+                        break;
+                    case VEHICLE:
+                        skillBonus.get(SkillType.S_GUN_VEE).setValue(1);
+                        skillBonus.get(SkillType.S_PILOT_GVEE).setValue(1);
+                        skillBonus.get(SkillType.S_PILOT_NVEE).setValue(1);
+                        skillBonus.get(SkillType.S_PILOT_VTOL).setValue(1);
+                        break;
+                }
```

Comment:
Forgot to add to this: Should there be bonuses here for the other phenotypes?

Reply:
I'm not familiar with the rules for it, so I can't say one way or the other. Shouldn't be too hard to implement if that's the case, though.